static/inline/css/mobile/host.css

```diff
@@ -0,0 +1,131 @@
+div[class^="tvp-"] {
+  box-sizing: border-box;
+}
+
+.tvp-modal-overlay {
```

Comment:
I think _Inline Template_ does not uses a modal, please remove all _tvp-modal_ css rules since we won;t use it if this is correct.